diff --git a/chrome/services/speech/BUILD.gn b/chrome/services/speech/BUILD.gn
index 97ccdc272c..96ff0a9b29 100644
--- a/chrome/services/speech/BUILD.gn
+++ b/chrome/services/speech/BUILD.gn
@@ -5,7 +5,8 @@
 import("//build/config/chromeos/ui_mode.gni")
 import("//chrome/services/speech/buildflags/buildflags.gni")
 
-assert(enable_speech_service)
+# Axium: Remove assert to allow building without speech_service
+# assert(enable_speech_service)
 
 source_set("lib") {
   sources = [
